cypress/integration/GlossaryAppIntegration/GlossaryAppIntegration.js

```diff
@@ -0,0 +1,152 @@
+/// <reference types="Cypress" />
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('user is on home page', () => {
+  cy.visit('/');
+});
+
+Given('user is on Spanish home page', () => {
+  cy.visit('/espanol');
+});
+
+When('user navigates to Dictionary tab', () => {
```

Comment:
I would not do this as part of the dictionaries. This is really the cancer.gov utility bar and not directly related to the dictionary.

Reply:
gone